biojava-structure/pom.xml

```diff
@@ -67,15 +67,16 @@
 			<version>1.4.0</version>
 		</dependency>
 
-		<!-- JAXb explicit dependency is needed since Java 9. See https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j
-		  Note versions are set in parent pom in dependency management section -->
 		<dependency>
-			<groupId>javax.xml.bind</groupId>
-			<artifactId>jaxb-api</artifactId>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>4.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.glassfish.jaxb</groupId>
 			<artifactId>jaxb-runtime</artifactId>
+			<version>4.0.3</version>
+			<scope>runtime</scope>
```

Comment:
Please remove version and scope here. It should be only in `dependencyManagement` in top pom.xml

Reply:
Removed explicit versions and scope from POM as required.